src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="calendar calendar--month-length-28">
+      <div class="calendar__date calendar__date--start-day-fri"></div>
```

Comment:
![image](https://github.com/mate-academy/layout_calendar/assets/99403198/5927bb38-2e43-4cc0-8819-bef40fe8096c)
```suggestion
    <div class="calendar calendar--month-length-28  calendar__date--start-day-fri">
      <div class="calendar__date"></div>
```